src/OT/glyf/SubsetGlyph.hh

```diff
@@ -55,6 +58,20 @@ struct SubsetGlyph
     return_trace (true);
   }
 
+  bool compile_bytes_with_deltas (const hb_subset_plan_t *plan,
+                                  hb_font_t *font,
+                                  const glyf_accelerator_t &glyf)
+  { return source_glyph.compile_bytes_with_deltas (plan, font, glyf, dest_start, dest_end); }
+
+  void free_compiled_bytes ()
+  {
+    if (dest_start.arrayZ)
```

Comment:
I believe it's safe to call `fini()` without checking `arrayZ`.

Reply:
I think we would need to check `arrayZ` cause the `fini()` for hb_bytes_t() doesn't check it.
In case of an empty glyph(e.g .notdef), the dest_start and dest_end would be empty hb_bytes_t(). And we do not want to free nullptr.

Reply:
Freeing nullptr is safe I believe.

Reply:
uh,...thank you! It's nullptr is not NULL